Vaibhav Shukla

```diff
@@ -0,0 +1 @@
+
```

Comment:
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>colors, border and backgrounds</title>
  <style>
    body {
      font-family: Arial, sans-serif;
      margin: 0;
      padding: 0;
    }

    header {
      background-color: green; 
      color: white;
      padding: 20px;
      text-align: center;
    }

    .content-box {
      border: 2px solid black;
      padding: 20px;
      margin: 20px;
      background-color: white;
    }

    section {
      background-image: url('https://via.placeholder.com/1200x600');
      background-size: cover; 
      background-position: center; 
      color: white;
      padding: 60px 20px;
      text-align: center;
    }

    section h2 {
      font-size: 36px;
      text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.5);
    }

    section p {
      font-size: 18px;
      margin-top: 20px;
    }
  </style>
</head>
<body>

  <header>
    <h1>Welcome to My Webpage</h1>
  </header>

  <div class="content-box">
    <h2>This is a div with a border</h2>
  </div>

  <section>
    <h2>Beautiful Background</h2>
  </section>

</body>
</html>